staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go

```diff
@@ -46,6 +48,16 @@ func ValidateLabelSelectorRequirement(sr metav1.LabelSelectorRequirement, fldPat
 		if len(sr.Values) > 0 {
 			allErrs = append(allErrs, field.Forbidden(fldPath.Child("values"), "may not be specified when `operator` is 'Exists' or 'DoesNotExist'"))
 		}
+	case metav1.LabelSelectorOpGt, metav1.LabelSelectorOpLt:
```

Comment:
This expands validation of all complex label selectors (replicasets, daemonsets, etc). Was that intended?